### PR TITLE
Don't focus on ItemsControl in PdbInfoViewer / PdbSourcesViewer

### LINKED ISSUE #1027: PdbInfo scroll box is centered instead of left
Looks like the control starts centered instead of left. There's also a brief pause/delay loading the tab and I'm not sure why. The data should already be parsed and available.

Can repro with package attacked to #1024 

image

@campersau any ideas?

### PR DESCRIPTION
**Pull request by campersau**
fixes #1027

### CODE REVIEW

File: PackageExplorer/Controls/PdbInfoViewer.xaml [hunk 1]
```diff
@@ -63,7 +63,7 @@
             Grid.Column="1"
             Visibility="{Binding CompilerFlags, Converter={StaticResource NullToVisibilityConverter}}"
             ItemsSource="{Binding CompilerFlags}"
-            MaxHeight="250">
```
Comment (campersau) [inline PackageExplorer/Controls/PdbInfoViewer.xaml:66]: @clairernovotny was there a reason why you did set a `MaxHeight`?

File: PackageExplorer/Controls/PdbInfoViewer.xaml [hunk 2]
```diff
@@ -35,7 +35,7 @@
             <RowDefinition Height="Auto" />
         </Grid.RowDefinitions>
 
-        <TextBox Text="Type:" Style="{StaticResource Key}" />
+        <TextBox Text="Type:" Style="{StaticResource Key}" IsTabStop="True" />
```
Comment (campersau) [inline PackageExplorer/Controls/PdbInfoViewer.xaml:38]: This is a workaround to not auto focus the first cell in the `DataGrid` which would scroll it into view...

File: PackageExplorer/Controls/PdbInfoViewer.xaml (hunk 1 above)
Comment (clairernovotny) [reply PackageExplorer/Controls/PdbInfoViewer.xaml:66]: The idea was that if the compiler list became too long, it'd scroll. That way the references wouldn't be too far down. Not sure on the best UX for the tab as a whole.

File: PackageExplorer/Controls/PdbInfoViewer.xaml (hunk 2 above)
Comment (clairernovotny) [reply PackageExplorer/Controls/PdbInfoViewer.xaml:38]: Lets just add a comment in the code so we remember why we did that down the road :)

Comment (campersau) [reply PackageExplorer/Controls/PdbInfoViewer.xaml:39]: Done.

**Pull request merged**